datastore/storagebackend/postgresql/postgresql.go

```diff
@@ -72,7 +72,7 @@ func NewPostgreSQL() (*PostgreSQL, error) {
 
 	host := common.Getenv("PGHOST", "localhost")
 	port := common.Getenv("PGPORT", "5433")
-	user := common.Getenv("PGUSER", "postgres")
+	user := common.Getenv("PGUSER", "db_user")
```

Comment:
datastore/README.md should be updated to reflect this new default value

Reply:
Changed it to `db_user`. Do you want me to place a comment somewhere that `db_user` is only accessible after the migrations and it has only CRUD privileges for the `data` database, and it is not a super user as the `postgres` user?

Reply:
Should it be updated in [this README.md](https://github.com/EURODEO/e-soh-datastore-poc/blob/main/datastore/README.md) too? Unsure ...

Reply:
I had my doubts about that one, but will wait for the other comment on the user: https://github.com/EURODEO/e-soh-datastore-poc/pull/76#discussion_r1422082402

Reply:
Removed the user, thus readme is back in its old state.